metrics.nim

```diff
@@ -1037,7 +1028,14 @@ when defined(metrics) and defined(linux):
     ticks = sysconf(SC_CLK_TCK).float64
     pagesize = sysconf(SC_PAGE_SIZE).float64
 
-  proc updateProcessInfo() =
+  type ProcessInfo = ref object of Gauge
+  var processInfo* {.global.} = ProcessInfo.buildCollector("process_info", "CPU and memory usage")
+
+  method collect*(collector: ProcessInfo): Metrics =
+    let timestamp = getTime().toMilliseconds()
+    result = initOrderedTable[Labels, seq[Metric]]()
```

Comment:
`initOrderedTable` is not needed since nim 0.20 (and overallocates memory for 64 entries)

Reply:
Removed.